layouts/404/single.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="p404-page page">
+    {{ partial "banner" .}}
+    <div  class="container-fluid">
+        <div class="container">
+        {{ partial "secondNav"  .}}
+        </div>
+        <div class="container-fluid pt-3 d-xs-block d-lg-none storj-blu-bg"></div>
```

Comment:
Don't need this one as well. It shows as blue block on mobile:

<img width="500" alt="blue" src="https://user-images.githubusercontent.com/3217669/81724728-2628f100-9485-11ea-93df-284ba48ecb5b.png">